vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datetimepicker/validation/BasicValidationIT.java

```diff
@@ -193,7 +211,7 @@ public void setValue_clearValue_assertValidity() {
 
     @Test
     public void badInput_changeValue_assertValidity() {
-        setInputValue(dateInput, "INVALID");
+        setFieldInvalid();
```

Comment:
Let's use explicit `setInputValue` everywhere to avoid potential side-effects.

Reply:
Updated to set the values explicitly.